src/transformers/models/tvp/processing_tvp.py

```diff
@@ -83,30 +109,28 @@ def __call__(self, text=None, videos=None, return_tensors=None, **kwargs):
             - **pixel_values** -- Pixel values to be fed to a model. Returned when `videos` is not `None`.
         """
 
-        max_text_length = kwargs.pop("max_text_length", None)
+        if "max_text_length" in kwargs:
+            kwargs["max_length"] = kwargs.pop("max_text_length")
 
         if text is None and videos is None:
             raise ValueError("You have to specify either text or videos. Both cannot be none.")
 
+        output_kwargs = self._merge_kwargs(
+            TvpProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
+            **kwargs,
+        )
+
         encoding = {}
         if text is not None:
-            textual_input = self.tokenizer.batch_encode_plus(
-                text,
-                truncation=True,
-                padding="max_length",
-                max_length=max_text_length,
-                pad_to_max_length=True,
-                return_tensors=return_tensors,
-                return_token_type_ids=False,
-                **kwargs,
-            )
+            textual_input = self.tokenizer(text, **output_kwargs["text_kwargs"])
```

Comment:
this actually doesn't match with the removed lines. Previously we had some kwargs hardcoded, like trucation or padding and now users can set those to any value.

TBH I don't know why these were hardcoded but I'd leave it as is to not break anything. In that case we don't need the `pad_to_max_length` kwarg, as it is going to be padded to `max-len` no matter what

Reply:
I moved them to the default values in `TvpProcessorKwargs`. I think that'd suffice since this part would've error-ed out anyway if downstream users passed them as kwargs too (duplicate args error).